src/config.py

```diff
@@ -155,6 +164,22 @@ def __init__(self, print_warnings: bool = False):
             "melee_min_score": int(self._select_val("claws", "melee_min_score")),
         }
 
+    def parse_item_config_string(self, key: str = None) -> ItemProps:
+        item_props = ItemProps()
+        # split string by commas NOT contained within parentheses
+        item_string_as_list = re.split(r',\s*(?![^()]*\))', self._select_val("items", key).upper())
```

Comment:
https://i.redd.it/an2lhgcr9kc61.png

Still love it : )

Reply:
isn't this the purpose of stack overflow? 

Also reminds me of this in my field of work: [:)](https://scontent-atl3-1.xx.fbcdn.net/v/t1.6435-9/79532782_1867938783341570_1908639344143368192_n.jpg?_nc_cat=100&ccb=1-5&_nc_sid=8bfeb9&_nc_ohc=1rJX9JU7mJsAX89TOhS&_nc_ht=scontent-atl3-1.xx&oh=00_AT_knVpqruEOvw3aN1qA0gVKiEcKE2nTdM81YupqdREyuw&oe=61E62CAB)